src/main/java/org/folio/converter/RecordToQuickMarcConverter.java

```diff
@@ -0,0 +1,84 @@
+package org.folio.converter;
+
+import static org.folio.converter.StringConstants.BLVL;
+import static org.folio.converter.StringConstants.CONTENT;
+import static org.folio.converter.StringConstants.TYPE;
+
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.io.IOUtils;
+import org.folio.rest.jaxrs.model.Field;
+import org.folio.rest.jaxrs.model.QuickMarcJson;
+import org.folio.srs.model.ParsedRecord;
+import org.folio.srs.model.Record;
+import org.marc4j.MarcJsonReader;
+import org.marc4j.MarcReader;
+import org.marc4j.marc.DataField;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Component
+public class RecordToQuickMarcConverter implements Converter<Record, QuickMarcJson> {
+  @Override
+  public QuickMarcJson convert(Record record) {
```

Comment:
As far as we will operate with ParsedRecord (https://github.com/folio-org/data-import-raml-storage/blob/master/schemas/dto/parsedRecordDto.json) it makes sense to change signature.